qhub/deploy.py

```diff
@@ -296,6 +296,8 @@ def provision_02_infrastructure(stage_outputs, config, check=True):
                 "kubeconfig_filename": os.path.join(
                     tempfile.gettempdir(), "QHUB_KUBECONFIG"
                 ),
+                "resource_group_name" : f'{config["project_name"]}-{config["namespace"]}',
```

Comment:
Just a comment for the future, it might be interesting to include those variable blocks somewhere else in the code, as @costrouc commented, maybe adding a .py file in the stage folder which contains those vars and importing in the relevant locations.